GVFS/GVFS.Common/Tracing/TelemetryDaemonEventListener.cs

```diff
@@ -91,48 +101,25 @@ private string SerializeMessage(TraceEventMessage message)
                 // Other TraceEventMessage properties are not used
             };
 
-            return pipeMessage.ToJson();
+            string json = pipeMessage.ToJson();
+
+            return Encoding.UTF8.GetBytes(json);

```

Comment:
We could do that, yes. I might want to then move this `QueuedPipeWriter` into the `Tracing` namespace though since it becomes a little less generic at that point. It is currently just a 'binary/byte' `*Writer'.